### ISSUE TITLE
Cannot pull image from cache mirror on 1.7: repository can only contain the runes

### ISSUE THREAD

**Issue by darthf1** [GoogleContainerTools/kaniko#1788]
**Actual behavior**
I'm getting the following on 1.7:

```
ERRO[0007] Error while retrieving image from cache: node:16.11.1-alpine repository can only contain the runes `abcdefghijklmnopqrstuvwxyz0123456789_-./`: docker-registry:5000 
```

**Expected behavior**
No error. the configuration works on 1.6.

**To Reproduce**
```
/kaniko/executor \
  --registry-mirror docker-registry:5000 \
  --insecure-pull \
  --cache \
  --context dir://client \
  --dockerfile ./client/Dockerfile \
  --destination ${CI_REGISTRY}/${CI_PROJECT_PATH}/client:${CI_COMMIT_SHORT_SHA}
```

**Comment by ergoz** [comment]
Same problem

**Comment by tdterry** [comment]
I just upgrade an old setup from 1.3 to 1.7, and this started happening. We run a registry mirror on :5003.

**Comment by imjasonh** [comment]
1.7 has some known regressions, can you try with latest (which is 1.6.0) or even better yet the latest commit tagged image?

Thats `:bde904349e9b7797d14f12c7a2657bf79145d18e`

**Comment by darthf1** [comment]
Resolved in v1.8
